app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt

```diff
@@ -77,15 +81,29 @@ class NotificationScheduler(
 
         when {
             (!variant().hasFeature(VariantManager.VariantFeature.RemoveDay1AndDay3Notifications) && privacyNotification.canShow()) -> {
-                scheduleNotification(OneTimeWorkRequestBuilder<PrivacyNotificationWorker>(), 1, TimeUnit.DAYS, UNUSED_APP_WORK_REQUEST_TAG)
+                val duration = getDurationForInactiveNotification(PRIVACY_DURATION)
+                scheduleNotification(OneTimeWorkRequestBuilder<PrivacyNotificationWorker>(), duration, TimeUnit.DAYS, UNUSED_APP_WORK_REQUEST_TAG)
             }
             (!variant().hasFeature(VariantManager.VariantFeature.RemoveDay1AndDay3Notifications) && clearDataNotification.canShow()) -> {
-                scheduleNotification(OneTimeWorkRequestBuilder<ClearDataNotificationWorker>(), 3, TimeUnit.DAYS, UNUSED_APP_WORK_REQUEST_TAG)
+                val duration = getDurationForInactiveNotification(CLEAR_DATA_DURATION)
+                scheduleNotification(OneTimeWorkRequestBuilder<ClearDataNotificationWorker>(), duration, TimeUnit.DAYS, UNUSED_APP_WORK_REQUEST_TAG)
             }
             else -> Timber.v("Notifications not enabled for this variant")
         }
     }
 
+    @VisibleForTesting
+    fun getDurationForInactiveNotification(day: Long): Long {
+        Timber.d("Inactive notification days installed is ${appInstallStore.daysInstalled()} day is $day")
+        var duration = day
+        if (variantHasInAppUsage() && (appInstallStore.daysInstalled() + day) == UOA_DURATION) {
```

Comment:
if the inactive notification clashes with the UOA notification on day 3 then add an extra day.